src/App.tsx

```diff
@@ -24,28 +24,28 @@ function App(_: Props): React.ReactElement {
   const [viewerID, setViewerID] = useState<string | null>(null);
 
   useEffect(() => {
-    console.log("listening to sockets");
-    socket.on("connect", () => {
-      console.log("connected");
+    console.log('listening to sockets');
+    socket.on('connect', () => {
+      console.log('connected');
     });
 
-    socket.on("disconnect", () => {
-      console.log("dcs");
+    socket.on('disconnect', () => {
+      console.log('dcs');
     });
 
-    socket.on("update_lobby", (lobby) => {
+    socket.on('update_lobby', (lobby) => {

```

Comment:
Any reason it's update_lobby instead of updateLobby? The rest of the code in other section uses camel casing while this is using an underscore

Reply:
this is intentional since it's not a variable, for routes and such, I use lower_snake_case instead of camelCase